NEWS.rst

```diff
@@ -2,6 +2,26 @@
 Release Notes
 =============
 
+pkgdev 0.2.1 (2022-05-21)
+-------------------------
+
+- pkgdev: Add configuration support. For more info look at [#]_.  (Arthur
+  Zamarin, #48, #62)
+
+- pkgdev commit: new summary for stabilizing ALLARCHES (Arthur Zamarin, #61)
+
+- pkgdev mask: offer to send last-rite message email to gentoo-dev ML when
+  last-riting a package (Arthur Zamarin, #63)
+
+- pkgdev manifest: add --if-modified (Arthur Zamarin, #66)
```

Comment:
I'd explain what this does, but you can leave that out if you want. I know people can use --help but it's nice to explain in release notes IMHO.

Ditto --ignore-fetch-restricted.

Reply:
Sounds good - thanks for the recommendation.